base/iterators.jl

```diff
@@ -1162,6 +1162,29 @@ end
 reverse(f::Flatten) = Flatten(reverse(itr) for itr in reverse(f.it))
 last(f::Flatten) = last(last(f.it))
 
+"""
+    Iterators.flatmap(f, iterators...)
+
+Equivalent to `flatten(map(f, iterators...))`.
+
+# Examples
+```jldoctest
+julia> Iterators.flatmap(n->-n:2:n, 1:3) |> collect
+9-element Vector{Int64}:
+ -1
+  1
+ -2
+  0
+  2
+ -3
+ -1
+  1
+  3
+```
+"""
+# flatmap = flatten ∘ map
```

Comment:
This line needs to be removed if you want the docstring to attach to the function. Should also be added to docs.

Reply:
I made a new PR #44905